docs/services/zwo_camera.rst

```diff
@@ -1,14 +1,145 @@
 ZWO Camera
 ==========
 
+This service operates a ZWO camera. There are currently four different types of ZWO cameras being used on the testbed:
+- ZWO ASI533MM (Science camera, https://www.zwoastro.com/product/asi533mm-mc/)
+- ZWO ASI290MM (TA camera, https://agenaastro.com/zwo-asi290mm-cmos-monochrome-astronomy-imaging-camera.html)
+- ZWO ASI178MM (MOWFS Camera and Pupil Camera, https://agenaastro.com/zwo-asi178mm-cmos-monochrome-astronomy-imaging-camera.html)
+- ZWO ASI1600MM (Phase Retrieval Camera, https://agenaastro.com/zwo-asi1600mm-p-cmos-monochrome-astronomy-imaging-camera-pro.html)
+
+For camera specs, see the website links above.
+
 Configuration
 -------------
 
+.. code-block:: YAML
+
+    science_camera:
+        service_type: zwo_camera
+        simulated_service_type: camera_sim
+        interface: hicat_camera
+        requires_safety: false
+
+        device_name: ZWO ASI533MM
+        offset_x: 1038
+        offset_y: 1282
+        width: 192
+        height: 192
+        well_depth_percentage_target: 0.65
+        exposure_time: 1000
+        gain: 100
+        exposure_time_step_size: 33
+        exposure_time_offset_correction: -79.2
+        exposure_time_base_step: 50
+
+    ta_camera:
```

Comment:
We only need one entry as an example. Remember, these are the catkit2 docs, not the hicat docs.